### PR TITLE
samba: fix build

### PR DESCRIPTION
**Pull request by natinusala**

### CODE REVIEW

File: packages/devel/netbsd-curses/package.mk [hunk 1]
```diff
@@ -28,8 +28,10 @@ PKG_LONGDESC="netbsd-libcurses portable edition"
 
 make_target() {
   make HOSTCC="$HOST_CC" PREFIX=/usr all-static
+  make HOSTCC="$HOST_CC" CC="$CC" PREFIX=/usr all
 }
 
 makeinstall_target() {
   make HOSTCC="$HOST_CC" PREFIX=$SYSROOT_PREFIX/usr install-static
+  make HOSTCC="$HOST_CC" CC="$CC" PREFIX=$INSTALL/usr install
```
Comment (ToKe79) [inline packages/devel/netbsd-curses/package.mk]: we need .so in SYSROOT for linking
```
+ make HOSTCC="$HOST_CC" PREFIX=$SYSROOT_PREFIX/usr intall
```

**Comment by ToKe79** [comment]
Hi,
this PR (e78362a236b7602b0aa80bf6261513152253e866) did not build correctly for all targets.
Following targets failed (1):
- Generic.x86_64

Logs and images for testing can be found here: http://nightly.builds.lakka.tv/PRs/pr_695_e78362a/

_This is an automated message_

**Comment by natinusala** [comment]
http://nightly.builds.lakka.tv/PRs/pr_695_e78362a/_logs/Generic.x86_64-pr_695_000025.txt

What kind of logs is that.....?

**Comment by plaidman** [comment]
🆙

**Comment by natinusala** [comment]
It looks like the readline SO wasn't properly cleaned / is corrupt...?

Le ven. 28 juin 2019 à 16:15, Jason Tomsic <notifications@github.com> a
écrit :

**Comment by ToKe79** [comment]
after manually removing `build.Lakka-Generic.x86_64/toolchain/lib/libreadline.so*` build was successful (`scripts/clean pkg_name` removes only the buildfolder/stams, does not remove files from toolchain/sysroot).

**Comment by natinusala** [comment]
I think #696 is a better version of this PR

**Comment by ToKe79** [comment]
Hi,
the build of this PR (e78362a236b7602b0aa80bf6261513152253e866) finished successfully for all 46 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_695_e78362a/

_This is an automated message_

**Pull request merged**